wasm2c/wasm-rt-impl.c

```diff
@@ -238,11 +238,14 @@ static void os_disable_and_deallocate_altstack(void) {
 #endif
 
 #if WASM_RT_USE_SEGUE && defined(__FreeBSD__)
-static void call_cpuid(uint64_t *rax, uint64_t *rbx, uint64_t *rcx, uint64_t *rdx) {
-  __asm__ volatile (
-    "cpuid"
-    : "=a" (*rax), "=b" (*rbx), "=c" (*rcx), "=d" (*rdx) // output operands
-    : "a" (*rax), "c" (*rcx) // input operands
+static void call_cpuid(uint64_t* rax,
+                       uint64_t* rbx,
+                       uint64_t* rcx,
+                       uint64_t* rdx) {
+  __asm__ volatile(
+      "cpuid"
+      : "=a"(*rax), "=b"(*rbx), "=c"(*rcx), "=d"(*rdx)  // output operands
+      : "a"(*rax), "c"(*rcx)                            // input operands
```

Comment:
This change (and the above one) look like maybe just reformatting changes?  Can they be split out or reverted?  (I'm sure you know, but you can use `git clang-format` to format just the touched parts of a file)

Reply:
Yup, this is already split out into #2619. It just shows up here since this PR is rebased on that one.